src/pages/PreviewScreen.jsx

```diff
@@ -4,31 +4,51 @@ import { Preview } from "@100mslive/hms-video-react";
 import { AppContext } from "../store/AppContext";
 import getToken from "../services/tokenService";
 
-const PreviewScreen = () => {
+const PreviewScreen = ({ getUserToken }) => {
```

Comment:
do we need to pass this getUserToken as a prop? can we not directly use it from util here itself?